ui/src/shared/components/ColorDropdown.tsx

```diff
@@ -7,28 +7,41 @@ import {Dropdown, ComponentStatus, DropdownMenuColors} from 'src/clockface'
 // Types
 import {ColorLabel} from 'src/types/colors'
 
-interface Props {
+interface PassedProps {
   selected: ColorLabel
-  disabled?: boolean
-  stretchToFit?: boolean
   colors: ColorLabel[]
   onChoose: (colors: ColorLabel) => void
 }
 
+interface DefaultProps {
+  disabled?: boolean
+  stretchToFit?: boolean
+  widthPixels?: number
+}
+
+type Props = PassedProps & DefaultProps
```

Comment:
ooh I really like this distinction actually

Reply:
Yarr Iris and Del gave me the idea